WordPress/Classes/Stores/StatsPeriodCache.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+final class StatsPediodCache {
+    static let shared = StatsPediodCache()
+
+    private var cache: [CacheKey: Any] = [:]
```

Comment:
Similar to the above comment, can this type be `[CacheKey: StatsTimeIntervalData]`?

Reply:
Updated